docs/README.md

```diff
@@ -43,6 +43,35 @@ It's also necessary to add a `type="text/language-name"` or `lang="language-name
 </style>
 ```
 
+## Documenting components
+
+To add documentation on a Svelte component that will show up as a docstring in
+LSP-compatible editors, you can use an HTML comment with the `@component` tag:
+
+```svelte
```

Comment:
Would be great if Github would support svelte syntax, but it doesn't as if now ^^ 

```suggestion
```html
```

Reply:
Awh. Alright, pushed it again :)